image/service/slapd/startup.sh

```diff
@@ -18,6 +18,10 @@ ulimit -n $LDAP_NOFILE
 chown -R openldap:openldap /var/lib/ldap
 chown -R openldap:openldap /etc/ldap
 chown -R openldap:openldap ${CONTAINER_SERVICE_DIR}/slapd
+chown openldap:openldap /etc/sasldb2
+
+# add sasldb credentials
+echo -n "$LDAP_ADMIN_PASSWORD" | saslpasswd2 -p -u "$LDAP_DOMAIN" -c admin
```

Comment:
warning LDAP_ADMIN_PASSWORD must be set when boostratping a new db but can be omited or empty later on